judgels-frontends/raphael/src/components/ClarificationFilterWidget/ClarificationFilterForm.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, Intent } from '@blueprintjs/core';
+import * as React from 'react';
+import { Field, InjectedFormProps, reduxForm } from 'redux-form';
+
+import { Required } from '../../components/forms/validations';
+import { FormSelect2 } from '../../components/forms/FormSelect2/FormSelect2';
+
+import './ClarificationFilterForm.css';
+
+export interface ClarificationFilterFormData {
+  status: string;
+}
+
+export interface ClarificationFilterFormProps extends InjectedFormProps<ClarificationFilterFormData> {
+  statuses?: string[];
+  isLoading: boolean;
+}
+
+const ClarificationFilterForm = (props: ClarificationFilterFormProps) => {
+  const { statuses } = props;
+  const statusField: any = statuses && {
```

Comment:
Can all occurrences of `statuses &&` in this PR can be removed? Because we always pass this props, so this is redundant.

Reply:
ok will update